AstraEngine.RaylibAdapter/RaylibCanvas2D.cs

```diff
@@ -77,13 +82,11 @@ public void DrawCircle(Position2D position, float radius, Color color)
             Raylib.DrawLineV(current, next, color.ToRayColor());
         }
     }
-    ///<inheritdoc/>
-    public void Drawtext(Position2D topleft, double fsize, Color color, string text)
-    {
 
-        // produces a lot of errors
-        Vector2 vector = new((float)topleft.X, (float)topleft.Y);
-        Raylib_cs.Raylib.DrawTextCodepoint(s_font, Enum.Parse(typeof(Alphabet), text).GetHashCode(), vector, (int)fsize, color.ToRayColor());
+    /// <inheritdoc/>
+    public void DrawSprite(string fileName, Position2D topLeft, Color tint)
+    {
+        Raylib.DrawTextureV(Raylib.LoadTexture(fileName), new Vector2((float)topLeft.X, (float)topLeft.Y), tint.ToRayColor());

```

Comment:
Does load texture reload the file every time you call this method? If so, we should not be doing this.

Reply:
How should I do this, as Sprite2D is not specifically Raylib so I can't use Texture2D in them, and I don't think I should cache the textures in RaylibTexture2D. Should I make a wrapper for Texture2D?